run.sh

```diff
@@ -65,14 +65,14 @@ done
 GALAXY_VIRTUAL_ENV="${GALAXY_VIRTUAL_ENV:-.venv}"
 if [ -d "$GALAXY_VIRTUAL_ENV" -a -z "$skip_venv" ];
 then
-    [ -n "$PYTHONPATH" ] && { echo 'Unsetting $PYTHONPATH'; unset PYTHONPATH; }
```

Comment:
I mostly agree, it was just the easiest way to ignore [SC2016 shellcheck warnings](https://github.com/koalaman/shellcheck/wiki/SC2016). On the other hand, explicit is better than implicit...

If you want I can revert them.